tests/utils/meilisearch-test-utils.ts

```diff
@@ -144,7 +144,7 @@ const datasetWithNests = [
   { id: 7, title: "The Hitchhiker's Guide to the Galaxy" },
 ];
 
-const dataset = [
+const dataset: { id: number; title: string; comment?: string }[] = [
```

Comment:
Can you fix the linter warning? 🙏 
<img width="492" alt="Capture d’écran 2024-08-21 à 10 46 17" src="https://github.com/user-attachments/assets/f19d618d-35c7-4e53-a5e9-605e292a5dfe">